### ISSUE TITLE
Waydroid does not start on Ubuntu 22.04 LTS with Radeon hardware

### ISSUE THREAD

**Issue by fir3-1ce** [waydroid/waydroid#835]
**Describe the bug**
Despite following the instructions in the documentation, Waydroid simply will not do anything when I launch the application in the Unity dash. (default Ubuntu Jammy with GNOME) I have absolutely no idea what the problem could be.

**Screenshots**
**INTERESTING FIND**
A user in a different issue thread said that locking the screen then reopening it made Waydroid work. This is what happened when I tried that, which makes me think it's a driver issue (even though the devs said they support all AMD hardware now):

![Screenshot from 2023-03-28 20-09-48](https://user-images.githubusercontent.com/113863279/228418171-e9d5be0b-85d2-4795-a11f-c0fd3f0d6479.png)

When I tried changing my `ro.hardware.gralloc=gbm` to `default` in waydroid_base.prop, it would show just a black screen that would immediately close then reopen when I clicked on it.

Sometimes, the Waydroid icon shows up in the Unity launcher panel (far left) then disappears as soon as I click on it.

Waydroid still shows that it's running in my System Monitor programs list, even though I see nothing. Mouse movements are kind of jerky too, indicating that there's some CPU load going on.

**General information (please complete the following information):**
 - `waydroid -V` = 1.4.0

`system_datetime = 1679116647
vendor_datetime = 1679138482
`
used Vanilla install

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04 LTS "Jammy"
 - GPU: AMD ATI Radeon RX 5500
 - Kernel version: `5.19.0-1021-lowlatency`
 - Host mesa version: mesa 22.0.5
 - Desktop: GNOME 42.5

**Additional context**
Tried uninstalling/reinstalling in case I did something wrong. Problem persists. Followed instructions on official website accurately. Ubuntu install is still fairly new, not too much tweaking that could have complicated things.

**Logs (please upload as file)**

[waydroid.prop.log](https://github.com/waydroid/waydroid/files/11096353/waydroid.prop.log)

[waydroid.log](https://github.com/waydroid/waydroid/files/11096392/waydroid.log)

waydroid logcat appeared to be endless, how long did you want me to let it run for? let it run for a while then forced `sudo killall waydroid` once it got to around 10 MB:

[logcat.txt](https://github.com/waydroid/waydroid/files/11096407/logcat.txt)

[dmesg.txt](https://github.com/waydroid/waydroid/files/11096408/dmesg.txt)

**Comment by Sideness** [comment]
Same problem here. Nothing happens when launching Waydroid. Server and session are running but nothing appears.

**Comment by not-mynumbness** [comment]
### I am experiencing a very similar issue. I got it to start but when the show-full-ui opens up it's all just garbled video output. And this is the only issue i've seen it brought up. I can't find any other information about this.

**Comment by fir3-1ce** [comment]
What did you do specifically? So far I ran the script and Waydroid still doesn't open.

**Comment by fir3-1ce** [comment]
HOLD THE PHONE

editing waydroid_base.prop to the specifications listed above works. it's kind of slow/laggy on my 4K monitor, but it appears to be working

**Comment by not-mynumbness** [comment]
> HOLD THE PHONE
> 
> editing waydroid_base.prop to the specifications listed above works. it's kind of slow/laggy on my 4K monitor, but it appears to be working

Yeah I found moding that worked out of the graphic glitching we had. I am running Intel UHD on a laptop with an AMD RX 580 via m.2 in a dual graphics. That helped with it atleast performing properly. Heh it’s still slow for me too though but it was 1% better than before. If you figure out any other tweaks to up performance or get better results keep me in the loop! So glad we got it this far.

**Comment by fir3-1ce** [comment]
unrelated to this thread, but do you know how to resize windows? or set to portrait view instead of landscape?

**Comment by not-mynumbness** [comment]
The best I got out of that was key:f11

**Comment by fir3-1ce** [comment]
i found something very helpful from their website: https://github.com/axel358/Waydroid-Settings

runs in a window now

**Comment by alej0varas** [comment]
Maybe it's related to #2160 where the solution that worked for me is https://github.com/waydroid/waydroid/issues/2160#issuecomment-3767826178
